packages/mobile/src/identity/contactMapping.ts

```diff
@@ -36,7 +36,8 @@ export function* doImportContacts() {
   try {
     yield call(getConnectedAccount)
 
-    const result: boolean = yield call(requestContactsPermission)
```

Comment:
I changed it to actually request contact perms because we now have a feature on the send + invite screens where if you don't have contacts imported yet, we show a button to import them (the evenly contractors added it a week or two ago).
The simplest way to make that work was to have this saga request the permissions.
Why is this saga triggered at all when the user presses skip? It seems like that's what we should fix

Reply:
The "Get my contacts" button on the Send + invite screen actually calls the `requestContactsPermission` directly from `src/utils/androidPermissions`. So this change shouldn't have an effect on it.

As for the saga being triggered when the user presses skip, I'll look into it.